presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -98,6 +111,10 @@ def _remove_conflicts_and_get_text_manipulation_data(
         2. Have the same indices as other results but with larger score.
         :return: List
         """
+        # Return without modifying analyzer_results.
+        if conflict_resolution == conflict_resolution.NONE:
+            return analyzer_results
+
         tmp_analyzer_results = []
         # This list contains all elements which we need to check a single result
```

Comment:
Should we put this under `ConflightResolutionStrategy.MERGE_SIMILAR_OR_CONTAINED`?

Reply:
Another approach would be to remove "None" from being one of the option. Otherwise there's no difference between MERGE_SIMILAR_AND_CONTAINED and None

Reply:
Hi @omri374 
-> Actually could you please explain it little bit more. Like, under ```ConflightResolutionStrategy.MERGE_SIMILAR_OR_CONTAINED``` means inside the code logic of MERGE_SIMILAR_OR_CONTAINED by setting some conditions. If this way then this can be possible but it will unnecessary increase the time complexity for None case. 
-> Are you talking about the "None" case that is handled below:
```
if not conflict_resolution:
            conflict_resolution = ConflictResolutionStrategy.MERGE_SIMILAR_OR_CONTAINED
```
This can be removed we can set None for ConflictResolutionStrategy.NONE case place as below
```
if (not conflict_resolution or conflict_resolution == conflict_resolution.NONE):
            return analyzer_results
```
If its different from what I can understand please elaborate little more. 

-> Although in test cases result looks similar for the options ```ConflictResolutionStrategy.MERGE_SIMILAR_OR_CONTAINED``` & ```ConflictResolutionStrategy.NONE```. But, for multiple entities result will differ. I think removing ```ConflictResolutionStrategy.NONE``` option itself is not best. 

Please let me know your thoughts on it.

Reply:
Hi @omri374 
Any comments on above approach.

Reply:
Hi @VMD7, apologies for the delay. I'll make sure to review this in the next few days.

Reply:
In high level, I see three scenarios:
1. What we have today (which I called `MERGE_SIMILAR_OR_CONTAINED`
2. The new strategy (what we call `REMOVE_INTERSECTIONS`)
3. Do nothing- no conflict handling at all. (what I think of when I see `None`). This might not make sense, and we can decide to skip it. 

My proposal is:
1. Put the existing logic under `MERGE_SIMILAR_OR_CONTAINED`)
2. Change the default to `MERGE_SIMILAR_OR_CONTAINED`)
3. Put the new logic under `REMOVE_INTERSECTIONS`)

I assume that `REMOVE_INTERSECTIONS` would need the current logic to run beforehand, right? If yes, then the if/else should represent this.

Hope this makes sense, apologies if I'm creating confusion here. We can also hop on another call if it helps.

Reply:
I'm not sure what you mean by "If this way then this can be possible but it will unnecessary increase the time complexity for None case."

Maybe the position of my comment was misleading. I was putting the comment with regards to the logic in line 122 (`other_elements = analyzer_results.copy()`)

Reply:
Hi @omri374 
Removed the option of NONE from conflict resolution strategy and the default is  same as ```MERGE_SIMILAR_OR_CONTAINED```. Please let me know your thoughts on it.

Reply:
Thanks @VMD7! I really appreciate all the work. I added one tiny comment and if you agree we can merge this.